fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/common/ThriftServer.java

```diff
@@ -35,7 +35,7 @@
  */
 public class ThriftServer {
     private static final Logger           LOG  = Logger.getLogger(ThriftServer.class);
-    private              ThriftServerType type = ThriftServerType.THREAD_POOL;
+    private              ThriftServerType type = ThriftServerType.THREADED_SELECTOR;
```

Comment:
maybe we can make it configurable for different scenario?

Reply:
sure. done.